app/src/main/java/de/danoeh/antennapod/preferences/PreferenceController.java

```diff
@@ -641,7 +654,7 @@ private void setUpdateIntervalText() {
                 val = String.format(context.getString(R.string.pref_autoUpdateIntervallOrTime_at),
                         timeOfDayStr);
             } else {
-                val = context.getString(R.string.pref_smart_mark_as_played_disabled);
+                val = context.getString(R.string.pref_smart_mark_as_played_disabled);  // TODO: Is this a bug? Otherwise document why is this related to smart mark???
```

Comment:
How about we change "pref_smart_mark_as_played_disabled" to be a generic "pref_disabled" that would be used for any disabled preference?  

The pref_smart_mark_as_played_disabled  string is already being translated, this just makes the usage for any preference intentional and self documenting.